File: ProAgil-App/src/app/_models/Speaker.ts
import { SocialNetwork } from './SocialNetwork';
import { Event } from './Event';

export interface Speaker {
    id: number; 
    name: string; 
    resumeCurriculum: string;
    imageURL: string;
    telephone: string;
    email: string;
    socialNetworks: SocialNetwork[];
    speakersEvents: Event[];
}

File: ProAgil.Domain/Speaker.cs
using System.Collections.Generic;

namespace ProAgil.Domain
{
    public class Speaker
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string ResumeCurriculum { get; set; }
        public string ImageURL {get; set;}
        public string Telephone { get; set; }
        public string Email { get; set; }
        public List<SocialNetwork> SocialNetworks { get; set; }
        public List<SpeakerEvent> SpeakersEvents { get; set; }
    }
}